test/routes/permitHolder/permitHolderDetails.route.test.js

```diff
@@ -15,7 +15,7 @@ const { COOKIE_RESULT } = require('../../../src/constants')
 
 const routePath = '/permit-holder/details'
 const companyNumberPath = '/permit-holder/company/number'
-const permitHolderName = '/permit-holder/name'
+const permitHolderNameRoute = '/permit-holder/name'
```

Comment:
Possibly change to "permitHolderNamePath"

Reply:
Done!